README.md

```diff
@@ -173,6 +173,37 @@ const styles = StyleSheet.create({
 
 Aphrodite will ensure that the global `@font-face` rule for this font is only inserted once, no matter how many times it's referenced.
 
+# Descendant selectors
+
+Descendant rules (i.e. a rule like `.parent:hover .child { ... }` in CSS) are created by adding values to your styles indicating the styles to apply to the child. To distinguish the styles from normal CSS keys, the keys start with a `>>`, like `'>>child'`.
+
+```js
+const styles = StyleSheet.create({
+    parent: {
```

Comment:
Both in the original spec you described and in this documentation, it wasn't clear to me whether `parent` was just a style name or whether `parent` carries some special meaning. It seems to be the latter, but is worth clarifying and perhaps using a different name.